### PR TITLE
Add test for nameable class

### PR DESCRIPTION
**Pull request by Benawi**
### General requirements
- [x] Make sure that you used the correct Gitflow.
- [x] Make sure that you document your work in a professional way.
### Project requirements
- [x]  Add unit tests for all the methods in all the classes:
        - `Book`
        - `Person`
        - `Student`
        - `Teacher`
        - `Classroom`
        - `Rental`
        - `all classes` that you created while implementing the Decorator design pattern

### CODE REVIEW

File: spec/capitalize_decorator_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+describe CapitalizeDecorator do
+  let(:decorator) { CapitalizeDecorator.new(double('decorated object')) }
+
+  describe '#correct_name' do
+    it 'capitalizes the name returned by the decorated object' do
+      allow(decorator).to receive(:correct_name).and_return('Benawi')
+
+      expect(decorator.correct_name).to eq('Benawi')
+    end
```
Comment (elisha2kyakpo1) [inline spec/capitalize_decorator_spec.rb]: - [ ] Kindly make understand how you are testing an already capitalized word. I expected to see a name with a small letter to equal the capitalized same name.
- [ ] Kindly test the trimmer_decorator method as well.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Benawi, @ProgramKingAbel, @sahlminkok, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ All the tests pass👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benawi** [comment]
### Changes ♻️
- [x] Test an already capitalized word. 
- [x] Test the methods of trimmer_decorator class

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Benawi, @ProgramKingAbel, & @sahlminkok, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title and summary. ✔️
- Add test ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**